Formula/burst.rb

```diff
@@ -0,0 +1,55 @@
+class Burst < Formula
+  desc "Radix sort, lazy ranges and iterators, and more. Boost-like header-only library"
+  homepage "https://github.com/izvolov/burst"
+  url "https://github.com/izvolov/burst/archive/refs/tags/v3.1.0.tar.gz"
+  sha256 "12089244b7dbb26c3198f2ea5b80555050c38efe0a33ce4de4c1fff9b1e7532d"
+  license "BSL-1.0"
+  head "https://github.com/izvolov/burst.git", branch: "master"
+
+  depends_on "cmake" => :build
```

Comment:
```suggestion
  depends_on "cmake" => [:build, :test]
```
Your `test` block will fail without this.